setup.py

```diff
@@ -4,10 +4,6 @@
 import sys
 import warnings
 
-if sys.version_info[:3] < (3, 7):
-    raise RuntimeError("Python version >= 3.7 required.")
```

Comment:
Do we still need this as long as setup.py exists?

Reply:
No, setuptools will use the `python_requires` config from `pyproject.toml`.